uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Hook/ValidateForm/AdminFormOptionsValidation.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * Class CRM_HRCore_Hook_ValidateForm_AdminFormOptionsValidation
+ */
+class CRM_HRCore_Hook_ValidateForm_AdminFormOptionsValidation {
+
+  /**
+   * Determines what happens if the hook is handled. Basically the hook
+   * is only handled for option values of the 'toil_amounts' and 'leave_days_amounts'
+   * option group. It basically ensures that the value of the value field of the
+   * option values for these option groups are validated before submission and
+   * if there are errors the form will not be submitted.
+   *
+   * @param string $formName
+   * @param array $fields
+   * @param mixed $files
+   * @param object $form
+   * @param array $errors
+   */
+  public function handle($formName, &$fields, &$files, &$form, &$errors) {
+    if (!$this->shouldHandle($formName, $form)) {
+      return;
+    }
+
+    $this->validateFormValues($fields, $errors);
+  }
+
+  /**
+   * Checks if the hook should be handled.
+   *
+   * @param string $formName
+   * @param object $form
+   *
+   * @return bool
+   */
+  private function shouldHandle($formName, $form) {
+    if($formName === 'CRM_Admin_Form_Options' && $form->elementExists('value')) {
```

Comment:
space after `if`, also you could probably use the static `class` constant instead of a string classname